Assets/Scripts/Services/Pause/IPause.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace UI
+{
+    public interface IPause
+    {
+        public bool Paused
```

Comment:
remove word public, by default all interface members is public. 
And pls edit this property to "bool Paused { get; set; }" (on line )

Reply:
Done